discord_bot/events.py

```diff
@@ -0,0 +1,39 @@
+from enum import Enum
+
+
+class Events (Enum):
+    """
+    Events are based on the section "Events" from the Discord API:
+    https://discordapp.com/developers/docs/topics/gateway
+    """
+    READY = 'READY'
```

Comment:
You use double quotes everywhere else in your codebase, let's find and replace `'` with `"`. I believe the hotkey is `Ctrl+H` or `Ctrl+R`.